2019-11-26 18:59:23,137 (61482/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 18:59:23,138 (61482/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 18:59:23,195 (61482/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "test" does not exist\n\n'
2019-11-26 19:00:00,768 (62251/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:00:00,769 (62251/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:00:00,844 (62251/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "test" does not exist\n\n'
2019-11-26 19:00:06,841 (62292/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:00:06,841 (62292/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:00:06,848 (62292/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "jth" does not exist\n\n'
2019-11-26 19:00:23,848 (62485/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:00:23,849 (62485/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:00:23,904 (62485/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "test" does not exist\n\n'
2019-11-26 19:05:13,047 (64587/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:05:13,048 (64587/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:05:13,061 (64587/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
2019-11-26 19:05:28,137 (64823/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:05:28,138 (64823/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:21:01,483 (66715/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:21:01,487 (66715/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:21:01,504 (66715/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
2019-11-26 19:21:04,297 (66840/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:21:04,299 (66840/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:21:04,311 (66840/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
2019-11-26 19:21:13,605 (66964/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:21:13,609 (66964/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-11-26 19:24:00,868 (67347/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-11-26 19:24:00,870 (67347/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-12-20 10:27:39,769 (39740/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-12-20 10:27:39,770 (39740/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-12-20 10:27:40,034 (39740/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "test" does not exist\n\n'
2019-12-20 16:15:56,209 (55610/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-12-20 16:15:56,211 (55610/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-12-20 16:16:11,736 (55610/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: fe_sendauth: no password supplied\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 566, in connect\n    database, user, passwd, host, port, dsn, **kwargs\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "bname" does not exist\n\n'
2019-12-20 16:16:26,004 (55742/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-12-20 16:16:26,016 (55742/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-12-20 16:17:30,594 (55742/MainThread) pgcli.pgexecute ERROR - sql: 'select *', error: SyntaxError('SELECT * with no tables specified is not valid\nLINE 1: select *\n               ^\n')
2019-12-20 16:17:30,597 (55742/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 402, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 433, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: SELECT * with no tables specified is not valid\nLINE 1: select *\n               ^\n\n'
2019-12-20 16:28:30,219 (56734/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2019-12-20 16:28:30,219 (56734/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2019-12-20 16:28:36,225 (56734/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: fe_sendauth: no password supplied\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 566, in connect\n    database, user, passwd, host, port, dsn, **kwargs\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  password authentication failed for user "read"\nFATAL:  no pg_hba.conf entry for host "63.119.220.66", user "read", database "d7rn1590oohv3u", SSL off\n\n'
2020-01-21 17:17:24,873 (78333/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-01-21 17:17:24,874 (78333/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-01-21 17:21:10,561 (78781/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-01-21 17:21:10,562 (78781/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:13:54,105 (91072/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:13:54,106 (91072/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:13:54,111 (91072/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2020-03-11 14:15:27,427 (91846/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:15:27,427 (91846/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:15:27,431 (91846/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2020-03-11 14:17:15,014 (92570/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:17:15,015 (92570/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:17:45,202 (92903/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:17:45,203 (92903/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:55:50,418 (99080/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:55:50,419 (99080/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-11 14:56:55,780 (99820/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-11 14:56:55,780 (99820/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-21 19:54:07,977 (70433/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-03-21 19:54:07,978 (70433/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-03-21 19:54:07,985 (70433/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 569, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/main.py", line 556, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 215, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 267, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 126, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.1.1_2/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: could not connect to server: Connection refused\n\tIs the server running on host "localhost" (::1) and accepting\n\tTCP/IP connections on port 5432?\ncould not connect to server: Connection refused\n\tIs the server running on host "localhost" (127.0.0.1) and accepting\n\tTCP/IP connections on port 5432?\n\n'
2020-07-01 10:29:23,904 (24992/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2020-07-01 10:29:23,905 (24992/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2020-07-01 10:29:23,910 (24992/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0/libexec/lib/python3.8/site-packages/pgcli/main.py", line 562, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.0.0/libexec/lib/python3.8/site-packages/pgcli/main.py", line 549, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0/libexec/lib/python3.8/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0/libexec/lib/python3.8/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.0.0/libexec/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: No such file or directory\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
2021-01-14 16:48:05,468 (91505/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-14 16:48:05,469 (91505/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-14 16:48:05,476 (91505/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 562, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 549, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not connect to server: No such file or directory\n\tIs the server running locally and accepting\n\tconnections on Unix domain socket "/tmp/.s.PGSQL.5432"?\n\n'
2021-01-14 16:48:28,657 (91680/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-14 16:48:28,658 (91680/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-14 16:48:28,788 (91680/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 562, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 549, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "test" does not exist\n\n'
2021-01-14 16:48:33,884 (91834/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-14 16:48:33,884 (91834/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-14 16:48:33,897 (91834/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 562, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 549, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "th" does not exist\n\n'
2021-01-14 16:48:43,486 (91992/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-14 16:48:43,486 (91992/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-14 16:48:43,499 (91992/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 562, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 549, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "th" does not exist\n\n'
2021-01-14 16:48:59,586 (92133/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-14 16:48:59,586 (92133/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-14 16:49:02,446 (92133/MainThread) pgcli.pgexecute ERROR - sql: '/d', error: SyntaxError('syntax error at or near "/"\nLINE 1: /d\n        ^\n')
2021-01-14 16:49:02,447 (92133/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "/"\nLINE 1: /d\n        ^\n\n'
2021-01-14 16:52:29,151 (92133/MainThread) pgcli.pgexecute ERROR - sql: '/d connections', error: SyntaxError('syntax error at or near "/"\nLINE 1: /d connections\n        ^\n')
2021-01-14 16:52:29,152 (92133/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 409, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 444, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.errors.SyntaxError: syntax error at or near "/"\nLINE 1: /d connections\n        ^\n\n'
2021-01-21 18:12:00,936 (15931/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-01-21 18:12:00,937 (15931/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-01-29 14:11:32,657 (15931/MainThread) pgcli.pgexecute ERROR - sql: '\\d profiles', error: OperationalError('server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n')
2021-01-29 14:11:32,662 (15931/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgspecial/dbcommands.py", line 711, in describe_table_details\n    cur.execute(sql)\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-01-29 14:11:32,662 (15931/MainThread) pgcli.main ERROR - sql: '\\d profiles', error: OperationalError('server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n')
2021-01-29 14:11:32,663 (15931/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 634, in execute_command\n    output, query = self._evaluate_command(text)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/main.py", line 880, in _evaluate_command\n    for title, cur, headers, status, sql, success, is_special in res:\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 398, in run\n    for result in pgspecial.execute(cur, sql):\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgspecial/main.py", line 90, in execute\n    return special_cmd.handler(cur=cur, pattern=pattern, verbose=verbose)\n  File "/usr/local/Cellar/pgcli/3.0.0_1/libexec/lib/python3.9/site-packages/pgspecial/dbcommands.py", line 711, in describe_table_details\n    cur.execute(sql)\npsycopg2.OperationalError: server closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n'
2021-02-07 13:09:28,645 (72978/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-07 13:09:28,646 (72978/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-07 13:09:28,706 (72978/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\npsycopg2.OperationalError: could not translate host name "curltail_development" to address: nodename nor servname provided, or not known\n\n'
2021-02-07 13:09:36,755 (73126/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-07 13:09:36,755 (73126/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-07 13:09:36,840 (73126/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "th" does not exist\n\n'
2021-02-08 13:27:44,456 (76415/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-08 13:27:44,457 (76415/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-08 13:27:44,530 (76415/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  role "postgres" does not exist\n\n'
2021-02-09 14:59:42,290 (68195/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-09 14:59:42,291 (68195/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-11 12:17:57,748 (13241/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-11 12:17:57,749 (13241/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:00:34,073 (80345/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:00:34,074 (80345/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:00:34,169 (80345/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "looseleaf_db_1" does not exist\n\n'
2021-02-17 13:00:58,666 (80535/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:00:58,666 (80535/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:00:58,714 (80535/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "th" does not exist\n\n'
2021-02-17 13:01:13,026 (80678/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:01:13,027 (80678/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:01:13,037 (80678/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 582, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/main.py", line 569, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 214, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 254, in connect\n    conn = psycopg2.connect(**conn_params)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/psycopg2/__init__.py", line 127, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/3.1.0/libexec/lib/python3.9/site-packages/pgcli/pgexecute.py", line 39, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "looseleaf_db_1" does not exist\n\n'
2021-02-17 13:11:37,481 (82894/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:11:37,482 (82894/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:17:15,807 (83858/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:17:15,808 (83858/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:34:57,358 (86286/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:34:57,360 (86286/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-17 13:35:52,463 (87181/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-17 13:35:52,464 (87181/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-02-23 14:11:03,950 (81793/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-02-23 14:11:03,951 (81793/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2021-03-03 14:50:36,130 (77607/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2021-03-03 14:50:36,131 (77607/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
